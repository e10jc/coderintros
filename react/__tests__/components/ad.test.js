import React from 'react'
import renderer from 'react-test-renderer'

import Ad from '../../components/ad'

test('renders', () => {
  const component = renderer.create(<Ad />)
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
