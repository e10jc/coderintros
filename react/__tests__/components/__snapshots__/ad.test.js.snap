// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div
  className="bg-silver  "
  style={
    Object {
      "height": undefined,
      "width": undefined,
    }
  }
>
  
</div>
`;
